packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -507,6 +599,12 @@ export default mixins(
 			expressionUpdated (value: string) {
 				this.valueChanged(value);
 			},
+			openExpressionEdit() {
+				if (this.isValueExpression) {
+					this.expressionEditDialogVisible = true;
+					return;
```

Comment:
let's omit the return here since no code is following

Reply:
changed